### ISSUE TITLE
missing namespace xml header.

### ISSUE THREAD

**Issue by toppk** [adobe/S3Mock#1754]
I feel like something obvious is wrong, but I cannot figure it out.  it seems l'm missing xmlns attributes in the response.

getting:
```xml
<ListBucketResult>...
```

expected:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ListBucketResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">...
```

**Comment by afranken** [comment]
@toppk S3Mock does indeed not always set the namespace in responses.
Not sure if S3 itself always does this either.

Do you need the namespace for anything? All S3 SDKs work without it...

**Comment by toppk** [comment]
it's been a while that I understand xml norms, but this tiny python library I'm using requires it.  (aiohttp-s3-client), which doesn't use a common s3 sdk.

I'm also a bit surprised that s3mock doesn't even send the `<?xml>` header.   I've tested minio/aws and they both send the namespace & xml header.   That being said, I'll look at this little library i'm using and see if it can be changed there.

**Comment by afranken** [comment]
S3Mock is implemented using the Spring Framework, which heavily favors the Jackson library.
Historically, Jackson is good for JSON and works OK for XML. 
For example, namespace declarations on the root element only (resulting in XML namespace defaulting) do not work:
https://github.com/FasterXML/jackson-dataformat-xml/issues/355

I'll look into the suggested workarounds.
Looks like adding the xml prolog just needs some configuration for Jackson's XmlMapper...

**Comment by afranken** [comment]
@toppk I released 3.6.0 yesterday which adds the XML prolog and namespace declarations to all responses. :)
